test/testUpCli.js

```diff
@@ -54,6 +55,12 @@ describe('hlx up', () => {
     sinon.assert.calledOnce(mockUp.run);
   });
 
+  it('hlx up can override secrets', () => {
+    new CLI()
+      .withCommandExecutor('up', mockUp)
+      .run('up --dev-default HTTP_TIMEOUT 2000');
+  });
```

Comment:
where is the assert that the param was set?

Reply:
Good point! Yeah I definitely duplicated logic. Thank you for pointing that out!